shared/src/commonMain/kotlin/io/customer/shared/util/DateTimeUtil.kt

```diff
@@ -8,7 +8,7 @@ internal interface DateTimeUtil {
     val nowUnixTimestamp: Long
 
     /**
-     * Returns unix timestamp in milliseconds.
+     * Returns unix timestamp in seconds.
```

Comment:
Reason for using seconds instead of milliseconds? 

For database operations, I would think that milliseconds is better as it's more accurate? For example: sorting in a database query. If you sort by `createdAt` with seconds, it's not as accurate of a sort compared to milliseconds.

Reply:
Agreed, that's why I kept `milliseconds` initially. But the server side is expecting `seconds` so I changed this. Although, I think we can still store `milliseconds` in database and convert to `seconds` before passing to the server, I'll check this and update accordingly.